spec/xcode_summary_spec.rb

```diff
@@ -51,6 +51,15 @@ module Danger
           ]
         end
 
+        it 'formats compile warnings with empty line' do
+          @xcode_summary.report('spec/fixtures/summary_with_empty_line.json')
+          expect(@dangerfile.status_report[:warnings]).to eq [
+            # rubocop:disable LineLength
+            "**<a href='https://github.com/diogot/danger-xcode_summary/blob/129jef029jf029fj2039fj203f92/Users/marcelofabri/Developer/MyAwesomeProject/MyAwesomeProject/Classes/AppDelegate.swift#L10001'>/Users/marcelofabri/Developer/MyAwesomeProject/MyAwesomeProject/Classes/AppDelegate.swift#L10001</a>**: File should contain 400 lines or less: currently contains 10001  <br />"
```

Comment:
Line `#L10001` is returned by SwiftLint JSON reporter and it's the last line of the file, which is usually empty